ui/src/shared/components/ConfirmButton.tsx

```diff
@@ -92,9 +84,27 @@ class ConfirmButton extends PureComponent<Props, State> {
     this.setState({expanded: false})
   }
 
-  private get calculatedPosition() {
+  private get className(): string {
+    const {type, size, square, disabled, customClass} = this.props
+    const {expanded} = this.state
+
+    const customClassString = customClass ? ` ${customClass}` : ''
+    const squareString = square ? ' btn-square' : ''
+    const expandedString = expanded ? ' active' : ''
+    const disabledString = disabled ? ' disabled' : ''
```

Comment:
Might be a good use for the `classnames` helper, à la https://github.com/influxdata/chronograf/blob/26c20183a5ca84fd76cd9580c12f04e034a0c988/ui/src/shared/components/threesizer/Division.tsx#L242-L249

Reply:
ahh good call, all those ternaries are hairy